lib/handlers/router-entities/route-caching/marshalling/pair-marshaller.ts

```diff
@@ -0,0 +1,26 @@
+import { Pair } from '@uniswap/v2-sdk'
+import { CurrencyAmountMarshaller, MarshalledCurrencyAmount } from './currency-amount-marshaller'
+import { Protocol } from '@uniswap/router-sdk'
+
+export interface MarshalledPair {
+  protocol: Protocol
+  currencyAmountA: MarshalledCurrencyAmount
+  tokenAmountB: MarshalledCurrencyAmount
```

Comment:
Why the different naming here between `currencyAmount` and `tokenAmount`

Reply:
that's the name that the `Pair` has in v2-sdk, so I just kept that consistent